app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt

```diff
@@ -8,7 +8,10 @@ import android.util.AttributeSet
 import android.util.TypedValue
 import android.view.View
 import androidx.core.content.ContextCompat
+import androidx.core.text.TextUtilsCompat
+import androidx.core.view.ViewCompat
 import org.oppia.android.R
+import java.util.*
```

Comment:
Wildcard import ktlint error
```suggestion
import java.util.Locale
```

Reply:
@FareesHussain  done